app/main.py

```diff
@@ -1 +1,28 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self, name: str, health: int = 100,
+                 hidden: bool = False) -> None:
+        self.health = health
+        self.name = name
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    def __repr__(self) -> str:
+        return (f"{{Name: {self.name}, Health: {self.health}, "
+                f"Hidden: {self.hidden}}}")
+
+
+class Herbivore(Animal):
+    def hide(self) -> None:
+        self.hidden = not self.hidden
+
+
+class Carnivore(Animal):
+    def bite(self, animal: Animal) -> None:
+        if (animal.hidden or animal not in animal.alive
+                or not isinstance(animal, Herbivore)):
+            return
+        animal.health -= 50
+        if animal.health <= 0:
+            Animal.alive.remove(animal)
```

Comment:
simplify this and get rid of return

Reply:
I changed it, but I'm not sure that it's better